index.js

```diff
@@ -17,6 +17,22 @@ module.exports.pitch = function (request) {
 
 	validateOptions(require('./options.json'), options, 'Style Loader')
 
+	var hmr = [
```

Comment:
Let's return the name `hmrBlock`. `hmr` is option, hmrBlock` is code block for `hmr`.

Reply:
The block doesn't add any semantic meaning, it already is an `{Array}`

Reply:
@michael-ciniawsky `hmrCode` or `hmrSource`?

Reply:
Why not just `hmr` as there isn't anything to distinguish from (atm). Only `options.hmr` (The Options) && `hmr` (The Code), The fact that it's a block (`{Array}` Type) and it is Source Code is already fulfilled 😛 , it just needs a name. `hmrCode` is also fine, but it's not really needed imho

Reply:
@michael-ciniawsky I think better using different variable names, even if they are stored in different places, it makes the reading easier not to be misleading, also when using destructuring it helps to avoid accidentally overriding the variable

Reply:
I'll just leave as is for now

Reply:
**if**, agreed. Still `const { code } = hmr` with `hmr` as the 'container/wrapper` **then** ;)

Reply:
hmm, `hmr` being a wrapper seems overkill here, but I do agree that it shouldn't be `hmr` to differentiate from the option's key, and this is why I put `hmrBlock` to indicate it's a code block. But as @michael-ciniawsky pointed out `block` being a bit confusing, I would say `hmrCode` is my pick. Updated.